src/layout/Header/index.tsx

```diff
@@ -13,60 +14,95 @@ const Header = () => {
+                <li>
+                  <a
+                    title="Home"
+                    href="#"
+                    className="flex pl-0 pr-[30px] font-Montserrat leading-[1.4em] tracking-[-0.04em] text-black no-underline"
+                  >
+                    Home
+                  </a>
+                </li>
+                <li>
+                  <a
+                    title="Textbook"
+                    href="#"
+                    className="flex pl-0 pr-[30px] font-Montserrat leading-[1.4em] tracking-[-0.04em] text-tk_greyDark no-underline"
+                  >
+                    Textbook
+                  </a>
+                </li>
+                <li>
+                  <a
+                    title="Statistics"
+                    href="#"
+                    className="flex pl-0 pr-[30px] font-Montserrat leading-[1.4em] tracking-[-0.04em] text-tk_greyDark no-underline"
+                  >
+                    Statistics
+                  </a>
+                </li>
+                <li
+                  className="relative"
+                  onMouseEnter={handleMouseEnter}
+                  onMouseLeave={handleMouseLeave}
+                >
+                  <a
+                    title="Games"
+                    href="#"
+                    className="flex pl-0 pr-[30px] font-Montserrat leading-[1.4em] tracking-[-0.04em] text-tk_greyDark no-underline"
+                  >
+                    Games
+                    <img src="/down_arrow.svg" alt="down array"></img>
+                  </a>
+                  {isHovered && (
+                    <div className=" absolute top-[46px] w-[215px] rounded-[14px] bg-white shadow-[0_343px_343px_0_rgba(23,64,51,0.08)] ">
+                      <DropDownMenu menuContent={['Sprint →', 'Audio-call →']} />
+                    </div>
+                  )}
+                </li>
+              </ul>
+            </div>
+          </nav>
+          <div className="flex gap-8 lg:hidden">
+            <LogInButton userName={'Alex'} isSimplified={true} />
+            <div className="flex md:hidden">
+              <MenuButton isMenuShown={isMenuShown} onClick={handleMouseClickOnMenuButton} />
+            </div>
+          </div>
+          <div className="hidden lg:flex">
+            <LogInButton userName={'Alex'} />
+          </div>
+        </div>
+      </header>
+      {isMenuShown && (
+        <div className="absolute top-[88px] z-10 flex h-[640px] w-full items-center justify-center bg-white md:hidden">
+          <ul className="text-center font-Dela_Gothic_One text-tk32 leading-8">
+            <li className="">Home</li>
+            <li className="my-[60px] text-tk_greyDark">Textbook</li>
+            <li className="my-[60px] text-tk_greyDark">Statistics</li>
+            <li className="flex justify-center text-tk_greyDark">
+              Games <img src="/down_arrow.svg" alt="down array"></img>
             </li>
```

Comment:
in mobile, the user clicks Games, the corresponding submenu should also be displayed

Reply:
https://github.com/taodemy/taotalk/blob/d8f609ef26d9a3dc290c100b53baff68a5dfc43a/src/layout/Header/index.tsx#L87-L98

The dropdown has also been added to links in mobile. The dropdown is set to "abstract" positioning mode and close to the link button. Then when mouse moves, the dropdown will not disappear.